README.md

```diff
@@ -1,67 +1,232 @@
+
+#### IDM
+
+This command is all about IDM configurations. It supports list, export, and import of IDM entities as well as additional commands available via the help argument. The command supports the same connection persistance we discussed above.
+
+##### Examples
+
+List IDM configuration objects. You'll notice that we don't need to specific the realm here and we're reading from existing connection info
+
+```shell
+frodo idm list <unique-connection-string>
+```
+
+Or by supplying the host, username, and password
+
+```shell
+frodo idm list https://<tenant-url>/am <username> <password>
+```
+
+###### Sample output
+
+```shell
+Listing all IDM configuration objects...
+ForgeRock Identity Cloud detected.
+Connected to ForgeRock Access Management 7.2.0-2021-11-SNAPSHOT Build 978ae0d483aa2da07826b3bdff286c60ccb41a4e (2022-February-09 11:40)
+- script
+- emailTemplate/frPasswordUpdated
+- emailTemplate/registration
+- secrets
+- servletfilter/payload
+- emailTemplate/baselineEmailVerification
+```
+
+Exports IDM configuration in JSON format to a file or `STDOUT` if you don't specify the file option. The `-N` option is the name of the configuration object
+
+```shell
+frodo idm export <unique-connection-string> -N <configuration-name> -f <file-name.json>
+```
+
+```shell
+frodo idm export https://<tenant-url>/am <username> <password> -N <configuration-name> -f <file-name.json>
+```
+
+#### Info
+
+`info` prints version information, session token, access token, and tokenId to `STDOUT`.
+
+##### Examples
+
+```shell
+frodo info <unique-connection-string>
+```
+
+```shell
+frodo info <https://idc.iam.example.com/am> <username> <password>
+```
+
+##### Sample output
+
+```shell
+Printing versions and tokens...
+ForgeRock Identity Cloud detected.
+Connected to ForgeRock Access Management 7.2.0-2021-11-SNAPSHOT Build 978ae0d483aa2da07826b3bdff286c60ccb41a4e (2022-February-09 11:40)
+Cookie name: <tenant-cookie-name>
+Session token: <your-session-token>
+Bearer token: <your-bearer-token>
+```
+
+#### Journey
+
+The journey command can list, export, import, prune and describe journeys in a specific realm. You can view all options and argument by using the `help` command. This command can export all journey in a given `realm`, specific journeys, or even all journeys as separate `JSON` files.
+
+#### Examples
+
+These examples assume a saved connection
+
+List all Journeys in a realm
+
+```shell
+frodo journey list <unique-connection-string> <realm-name>
+```
+
+Export a specific journey
+
+```shell
+frodo journey export <unique-connection-string> <realm-name> -t <journey-name>
+```
+
+Import a Journey from a `JSON` file
+
+```shell
+frodo journey import <unique-connection-name> <realm-name> -t <journey-name> -f <file-name>
+```
+
+#### Logging
+
+Todo
+
+## Developing
+
+### Prerequisites
+
 - Install nodejs (tested on v14.9.0) and npm (included with node)
```

Comment:
We're currently building the binaries on 16.13.x. I am running 16.13.0 on my mac. We may want to discuss what we claim to test.

Reply:
I use node 16 as well. I'll update, but we should discuss more. one of my test subjects ran into issues with node 17